blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java

```diff
@@ -1,76 +0,0 @@
-/*
```

Comment:
I think ScheduledContext is a real class that we still use. Maybe we shouldn't be removing the whole file, but only the parts that uses ZooKeeper?